### PR TITLE
Translate document "5 visualization" into Chinese

### PR DESCRIPTION
**Pull request by LihanChen2004**
Translate document "5 visualization" into Chinese

### CODE REVIEW

**Comment by JingweiZhang12** [comment]
@LihanChen2004 . Please open a new doc in docs/zh_cn rather than modify the English docs in place.

File: docs/en/user_guides/5_visualization.md [hunk 1]
```diff
@@ -1,67 +1,72 @@
...
+# 学习如何进行可视化操作
 
-## Local Visualization
+## 本地可视化
 
-This section will present how to visualize the detection/tracking results with local visualizer.
+本节将介绍如何使用本地可视化工具可视化检测/跟踪结果。
 
-If you want to draw prediction results, you can turn this feature on by setting `draw=True` in `TrackVisualizationHook` as follows.
+如果你想绘制预测结果，你可以通过在`TrackVisualizationHook`中设置`draw=True`打开这个功能，如下所示。
```
Comment (JingweiZhang12) [inline docs/en/user_guides/5_visualization.md:7]: Please note that the one blank space between English & numbers and Chinese and check all the contents.

File: docs/en/user_guides/5_visualization.md [hunk 2]
```diff
@@ -1,67 +1,72 @@
...
 
-We realize the detection visualization with class `DetLocalVisualizer`.
-You can call it as follows.
+我们将在下面详细介绍。
+
+关于[Visualization](https://github.com/open-mmlab/mmengine/blob/main/docs/en/advanced_tutorials/visualization.md)和[Hook](https://github.com/open-mmlab/mmengine/blob/main/docs/en/tutorials/hook.md)的更多细节，可以参考MMEngine。
+
+#### 可视化检测
+
+我们用类`DetLocalVisualizer`实现检测可视化。
```
Comment (JingweiZhang12) [inline docs/en/user_guides/5_visualization.md:33]: 类`DetLocalVisualizer` > `DetLocalVisualizer` 来

File: docs/en/user_guides/5_visualization.md [hunk 3]
```diff
@@ -1,67 +1,72 @@
...
 
-Here is a visualization example of DFF:
+下面是一个DFF的可视化示例：
 
 ![test_img_29](https://user-images.githubusercontent.com/99722489/186062793-623f6b1e-163e-4e1a-aa79-efea2d97a16d.png)
 
-#### Tracking Visualization
+#### 跟踪可视化
+
+我们通过类`TrackLocalVisualizer`实现跟踪可视化。
```
Comment (JingweiZhang12) [inline docs/en/user_guides/5_visualization.md:58]: 类`TrackLocalVisualizer` >  `TrackLocalVisualizer` 来

File: docs/en/user_guides/5_visualization.md [hunk 4]
```diff
@@ -1,67 +1,72 @@
...
 
-We realize the tracking visualization with class `TrackLocalVisualizer`.
-You can call it as follows.
+您可以这样调用它。
```
Comment (JingweiZhang12) [inline docs/en/user_guides/5_visualization.md:60]: 您可以通过这样增加 configs 配置来调用它。 Please update all the same sentences.

File: docs/en/user_guides/5_visualization.md (hunk 1 above)
Comment (LihanChen2004) [reply docs/en/user_guides/5_visualization.md:7]: Thanks for guidance. I have modified and submitted it to `dev-1.x`

**Pull request closed without merging**